wagtail/wagtailadmin/templates/wagtailadmin/base.html

```diff
@@ -37,3 +48,55 @@
         </div>
     </div>
 {% endblock %}
+
+{# Animation easter-egg: #}
+{% block extra_js %}
+    <script>
+        $(function() {
+          var sensitivity = 8; // the amount of times the user must stroke the wagtail to trigger the animation
+
+          var $logoContainer = $('.wagtail-logo-container__desktop');
```

Comment:
I don't think it's a big deal - jQuery is an existing dependency of the Wagtail admin that isn't likely to be going away any time soon. (Also, jQuery collections have the handy property that the code won't break if the logo element isn't there, e.g. because the logo has [been customised](http://docs.wagtail.io/en/v1.11.1/advanced_topics/customisation/admin_templates.html#custom-branding))